it/connectors/IntegrationFrameworkConnectorISpec.scala

```diff
@@ -66,6 +66,25 @@ class IntegrationFrameworkConnectorISpec extends ConnectorIntegrationTest
       await(underTest.getAllStateBenefitsData(taxYear, nino)(hc)) shouldBe
         Left(ApiError(INTERNAL_SERVER_ERROR, SingleErrorBody("some-code", "some-reason")))
     }
+
```

Comment:
Split the tests to clearly say that when 23-24 do that (you have this), when not 23-24 do something else (you don't have that.)